src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+
+
+public class IntakeSubsystem extends SubsystemBase {
+WPI_TalonSRX intakeDeployMotor;
+WPI_TalonSRX intakeRollerMotor;
+
+private DigitalInput intakeLimitSwitchU;
+private DigitalInput intakeLimitSwitchD;
```

Comment:
don't need to include intake in the name since they are In The Intake File
`limitSwitchUp` and `limitSwitchDown`